### PR TITLE
Issue#280: MUI | Send to bell members button functionality on survey detail page, Issue#281: MUI | Get members list button functionality, Issue#282: MUI | Send by criteria button functionality, Issue#283: MUI | Members survey page--Resolved By: Umar Waris, Submission Date: June 06, 2016

### PR DESCRIPTION
**Pull request by umarWaris**
https://waffle.io/open-learning-exchange/BeLL-Apps/cards/574fc4e6b285165f00178fd2
https://waffle.io/open-learning-exchange/BeLL-Apps/cards/574fc5158ed2891400b93449
https://waffle.io/open-learning-exchange/BeLL-Apps/cards/574fc581d26b7b7a00706c29
https://waffle.io/open-learning-exchange/BeLL-Apps/cards/574fc5a08ed2891400b9344a

### CODE REVIEW

File: app/nation/app/Router.js [hunk 1]
```diff
@@ -4004,12 +4004,18 @@ $(function() {
...
+            var lang = App.Router.getLanguage(loginOfMem);
+            App.languageDictValue=App.Router.loadLanguageDocs(lang);
             var membersListView = new App.Views.MembersListView();
             membersListView.surveyId = surveyId;
             membersListView.selectedBellCodes = selectedBellCodes;
             membersListView.selectedBellNames = selectedBellNames;
             membersListView.render();
-            App.$el.children('.body').html(membersListView.el);
+            App.$el.children('.body').html('<div id="memberList"></div>');
+            $('#memberList').append('<h3>' + App.languageDictValue.get('Surveys') + '</h3>');
```
Comment (MuhammadWaqas05) [inline app/nation/app/Router.js]: This heading is extra, please remove it.

File: app/nation/app/Home.css [hunk 2]
```diff
@@ -28,219 +35,219 @@ color: #345474;
...
+    text-indent:0;
+    border:2px solid #8A1732;
+    display:inline-block;
+    color:#fff;
+    font-family: verdana;
+    font-weight:bold;
+    font-style:normal;
+    text-decoration:none;
+    text-align:center;
+
```
Comment (SadiaRasheed) [inline app/nation/app/Home.css:212]: remove this empty line

File: app/nation/app/Home.css [hunk 3]
```diff
@@ -28,219 +35,219 @@ color: #345474;
...
+    /*text-shadow:1px 1px 12px #689324;*/
```
Comment (SadiaRasheed) [inline app/nation/app/Home.css:213]: This is unnecessary commented line, please remove this

File: app/nation/app/Home.css [hunk 4]
```diff
@@ -28,219 +35,219 @@ color: #345474;
...
 }
 
 .btn-green, .btn-red
 {
-	 height: 40px;
+    height: 40px;
     margin-left: 19px;
```
Comment (SadiaRasheed) [inline app/nation/app/Home.css:219]: please check the Indentation

File: app/nation/app/Home-Urdu.css [hunk 5]
```diff
@@ -39,6 +39,9 @@ h1,h2,h3,h4,h5,h6{
 #parentDiv{
     direction: rtl;
 }
+#sendToBellDiv{
```
Comment (SabaBaig) [inline app/nation/app/Home-Urdu.css:45]: Try to reuse styles that are already defined in .css

**Comment by SadiaRasheed** [comment]
I have added comments where it was necessary.

**Comment by SabaBaig** [comment]
Just 2 suggestions
1) Try to re-use the existing functionality so as to avoid redundant code snippets.
2) Avoid having such long commits involving only difference in spaces.

**Pull request merged**